kong.conf.default

```diff
@@ -1826,3 +1826,29 @@ G# -----------------------
 
 #wasm_filters_path =              # Path to the directory containing Wasm filters
                                   # that Kong must load on startup.
+
+#wasm_shm_kv =                    # Define a shared key/value memory zone accessible
+                                  # through all Contexts and by all nginx worker
+                                  # processes.
+
+#wasm_shm_queue =                 # Define a shared queue memory zone accessible
+                                  # through all nginx contexts and by all nginx
+                                  # worker processes.
+
+#wasm_tls_no_verify_warn =        # Toggle the TLS verification warning log for
+                                  # Wasm sockets. 
+
+#wasm_socket_buffer_reuse =       # Toggle Wasm sockets buffer reuse.
+
+#wasm_socket_buffer_size =        # Set a buffer `size` for reading response payloads
+                                  # with Wasm sockets.
```

Comment:
Ah, wasm sockets. Does it make sense to hide this right now and re-use the nginx buffer size?

Reply:
In the Gateway and as already pointed out in sync-up meetings, this option is indeedintended to receive the default (or specified) nginx buffer sizes.